informer/disk/disk.go

```diff
@@ -12,13 +12,21 @@ import (
 	"github.com/ipfs/ipfs-cluster/api"
 )
 
+// TODO: switch default to disk-freespace
+const DefaultMetric = "disk-reposize"
+
 var logger = logging.Logger("diskinfo")
 
 // MetricTTL specifies how long our reported metric is valid in seconds.
 var MetricTTL = 30
 
 // MetricName specifies the name of our metric
-var MetricName = "disk"
+var MetricName string
```

Comment:
Before this was more like a configuration option for the module, but now it's a property of the informer defined on creation, so it should not exist anymore as global variable.